rust/Cargo.toml

```diff
@@ -42,6 +42,9 @@ async-trait = "0.1.35"
 influxdb = { version = "0.1.0", features = ["derive"], optional = true }
 chrono = { version = "0.4.13", optional = true }
 
+[dependencies.xaynet-metrics-macro]
+path = "xaynet-metrics-macro"
```

Comment:
Not 100% sure but I think we need `optional = true`. Otherwise, will it not be always pulled in?

Reply:
Although I think we actually need it even without the `metrics` feature flag. So it's probably the line 
```toml
metrics = ["influxdb", "chrono", "xaynet-metrics-macro/metrics"]
``` that should be changed back to 
```toml
metrics = ["influxdb", "chrono"]
```
Correct?